setup.mk

```diff
@@ -30,9 +30,9 @@ export BASE:=$(CURDIR)
 
 # read license file and do reformatting for proper display
 export LICENSE_TEXT:=$(cat LICENSE_FILE)
-export LICENSE_TEXT_COLUMN:=$(fold -w 78 -s LICENSE_FILE)  # Format to 80 characters
-export LICENSE_TEXT_COMMENTED:=$(LICENSE_TEXT_COLUMN | sed -e 's/^/\# /g')
+export LICENSE_TEXT_COLUMN:=$(shell fold -w 78 -s $(LICENSE_FILE))  # Format to 80 characters
+export LICENSE_TEXT_COMMENTED:=$(shell echo $(LICENSE_TEXT_COLUMN) | sed  's/^/\# /g' )
```

Comment:
`/` seems to be treated as special in make?

using a different character seems to work for me `sed  's!^!\# !g'`